planet/api/__init__.py

```diff
@@ -40,4 +40,5 @@ class ClientV1DeprecationWarning(FutureWarning):
     "For more details please see the discussion at "
     "https://github.com/planetlabs/planet-client-python/discussions.",
     ClientV1DeprecationWarning,
+    stacklevel=2
```

Comment:
Now, the warning will point to the module that imports planet.api instead of to planet.api itself.